apps/tiltakskoordinator-flate/src/components/Kontaktinformasjon.tsx

```diff
@@ -0,0 +1,122 @@
+import { BodyLong, Heading } from '@navikt/ds-react'
+import { DeltakerDetaljer } from '../api/data/deltaker'
+import {
+  Buildings3Icon,
+  EnvelopeClosedIcon,
+  HouseIcon,
+  PersonIcon,
+  PhoneIcon
+} from '@navikt/aksel-icons'
+import { formatTelefonnummer } from '../utils/utils'
+
+interface Props {
+  deltaker: DeltakerDetaljer | null
+}
+
+export const Kontaktinformasjon = ({ deltaker }: Props) => {
+  if (!deltaker) {
+    return null
+  }
+
+  return (
+    <div className="w-fit">
+      <Box>
+        <>
+          <Heading level="3" size="xsmall">
+            Kontaktinformasjon
+          </Heading>
+          {deltaker.kontaktinformasjon.telefonnummer && (
+            <Item
+              icon={PhoneIcon}
+              title="Telefonnummer"
+              detail={formatTelefonnummer(
+                deltaker.kontaktinformasjon.telefonnummer
+              )}
+            />
+          )}
+          {deltaker.kontaktinformasjon.epost && (
+            <Item
+              icon={EnvelopeClosedIcon}
+              title="Epost"
+              detail={deltaker.kontaktinformasjon.epost}
+            />
+          )}
+          {deltaker.kontaktinformasjon.adresse && (
+            <Item
+              icon={HouseIcon}
+              title="Adresse"
+              detail={deltaker.kontaktinformasjon.adresse}
+            />
+          )}
+        </>
+      </Box>
+
+      <Box>
+        <>
+          <Heading level="3" size="xsmall">
+            NAV-kontor
```

Comment:
Sikkert noe å diskutere med Live om, men den offisielle skrivemåten for Nav er vel `Nav` nå, ikke allcaps.